### PR TITLE
Fixes minor docs errors

### PR DESCRIPTION
**Pull request by mvadari**
## High Level Overview of Change

This PR:
* Shows all the inherited members for amount/currency objects
* Fixes the use of `send_reliable_submission` in the README
* Fixes repeating content on the "XRP Global Variables" page

### Context of Change

Python SDK bugfixes

### Type of Change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (non-breaking change that only restructures code)
- [ ] Tests (You added tests for code that already exists, or your new feature included in this PR)
- [x] Documentation Updates
- [ ] Release

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -157,7 +157,7 @@ Use the [`xrpl.transaction`](https://xrpl-py.readthedocs.io/en/latest/source/xrp
...
 
-* [`send_reliable_submission`](https://xrpl-py.readthedocs.io/en/latest/source/xrpl.transaction.html#xrpl.transaction.send_reliable_submission) — An implementation of the [reliable transaction submission guidelines](https://xrpl.org/reliable-transaction-submission.html#reliable-transaction-submission), this method signs a transaction locally, submits the transaction to the XRP Ledger, and then verifies that it has been included in a validated ledger (or has failed to do so). Use this method to submit transactions for production purposes.
+* [`send_reliable_submission`](https://xrpl-py.readthedocs.io/en/latest/source/xrpl.transaction.html#xrpl.transaction.send_reliable_submission) — An implementation of the [reliable transaction submission guidelines](https://xrpl.org/reliable-transaction-submission.html#reliable-transaction-submission), this method submits a signed transaction to the XRP Ledger, and then verifies that it has been included in a validated ledger (or has failed to do so). Use this method to submit transactions for production purposes.
```
Comment (ryangyoung) [inline README.md:160]:
```suggestion
* [`send_reliable_submission`](https://xrpl-py.readthedocs.io/en/latest/source/xrpl.transaction.html#xrpl.transaction.send_reliable_submission) — An implementation of the [reliable transaction submission guidelines](https://xrpl.org/reliable-transaction-submission.html#reliable-transaction-submission), this method submits a signed transaction to the XRP Ledger and then verifies that it has been included in a validated ledger (or has failed to do so). Use this method to submit transactions for production purposes.
```

**Review by ryangyoung — APPROVED** [review]
👍 thank you for the fix!

**Comment by yyolk** [comment]
FYI #232 brought in d718cc86cee451481f0ad59164abc96936184230

a re-base will remove the cherry-picked commit and resolve conflicts for this MR :)

**Comment by mvadari** [comment]
Thanks for the tip!

**Review by ledhed2222 — APPROVED** [review]

**Pull request merged**